cmd/wasm-client/go.mod

```diff
@@ -5,12 +5,12 @@ require (
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/faraday v0.2.11-alpha
-	github.com/lightninglabs/lightning-node-connect v0.2.4-alpha
+	github.com/lightninglabs/lightning-node-connect v0.2.5-alpha
 	github.com/lightninglabs/lightning-terminal v0.10.5-alpha
 	github.com/lightninglabs/loop v0.26.2-beta
 	github.com/lightninglabs/pool v0.6.4-beta
-	github.com/lightninglabs/taproot-assets v0.2.3
-	github.com/lightningnetwork/lnd v0.16.4-beta
+	github.com/lightninglabs/taproot-assets v0.3.0
```

Comment:
Can we bump Loop to `v0.26.4-beta` and Litd to `v0.11.0` please?

Reply:
Ok, will do!